packages/dappmanager/test/integration/hostScripts.test.int.ts

```diff
@@ -25,4 +26,31 @@ describe("Host scripts", () => {
     );
     expect(hostInfo).to.be.ok;
   });
+
+  it("Should execute security updates", async () => {
+    try {
+      const securityUpdate = await shell(
+        `sudo bash ${hostScriptsPath}/security_update.sh`
```

Comment:
Can we find an alternative without sudo?

Reply:
https://github.com/dappnode/DNP_DAPPMANAGER/pull/748#discussion_r625614512